### PR TITLE
🚧🚧 upgraded minimum deployment version to iOS 13.0 🚧🚧

### PR DESCRIPTION
**Pull request by Climbatize**
# Description
Removed all version tests and deprecated iOS methods
🚧🚧 Don't merge before major version release has been approved 🚧🚧

## Links
[MINI-2665](https://jira.rakuten-it.com/jira/browse/MINI-2665)

# Checklist
- [X] I have read the [contributing guidelines](CONTRIBUTING.md).
- [X] I have completed the [SDK Development Learning Path](CONTRIBUTING.md#SDK-Development-Learning-Path) (if submitting a feature PR)
- [X] I wrote/updated tests for new/changed code
- [X] I removed all sensitive data before every commit, including API endpoints and keys
- [X] I ran `fastlane ci` without errors

### CODE REVIEW

**Review by corycaywood — COMMENTED** [review]
Looks good, but please also update the changelog with this.

File: Example/AppDelegate.swift [hunk 1]
```diff
@@ -15,12 +15,9 @@ class AppDelegate: UIResponder, UIApplicationDelegate {
...
     func application(_ application: UIApplication, supportedInterfaceOrientationsFor window: UIWindow?) -> UIInterfaceOrientationMask {
-        if #available(iOS 12, *) {
-            if window?.isKeyWindow != true {
-                return .all
-            }
-        }
-        if MiniApp.MAOrientationLock.isEmpty {
+        if window?.isKeyWindow != true {
+            return .all
+        } else if MiniApp.MAOrientationLock.isEmpty {
```
Comment (ts-leojoseph-r) [inline Example/AppDelegate.swift:20]: 😎

**Review by ts-leojoseph-r — APPROVED** [review]
Looks good to me :+1:

**Review by corycaywood — APPROVED** [review]
Looks good 👍

**Pull request merged**